### ISSUE TITLE
Resolve function in exec isn't called after action succeeded or failed (commandDelegate missing)

### ISSUE THREAD

**Issue by panczarny** [timbru31/cordova-plugin-lottie-splashscreen#134]
# Bug report

## What steps will reproduce the problem?
1. Use `cordova-plugin-lottie-splashscreen`
2. In your javascript write sth like:
```javascript
global.lottie.splashscreen
  .hide()
  .then(() => {})
  .catch(() => {});
```
3. The splashscreen gets hidden, but none of `then` or `catch` is processed.

## Expected output
The `Promise` gets resolved

## What do you see instead?
Non resolved `Promise`

## Log file of the issue/error
N/A

## Version information
N/A

## Please provide any additional information below.
As I did debug, the issue seems to be in the `LottieSplashScreen.swift`, where there is no `commandDelegate.send` action (check here https://medium.com/modus-create-front-end-development/writing-a-cordova-plugin-in-swift-3-for-ios-e314acf68a47). Unfortunately, I wasn't able to fix it on my own, as I'm not a Swift developer 😿.
I did
```swift
commandDelegate!.send(
  CDVCommandStatus_OK,
  callbackId: command.callbackId
)
```
but the code didn't compile

## Checklist

-   [X] If there is a (potential) plugin conflict, I've identified the conflicting plugin
-   [ ] I have added a valid version output
-   [X] I have attached necessary information like a screenshot, example project or videos

**Comment by timbru31** [comment]
Thanks, I'll debug this. Ironically we use it in production, too, to hide the animation via JS - but it seems no one has found out that the Promise is never resolved.

Is Android affected, too?

**Comment by timbru31** [comment]
No, Android has success and error callbacks.

**Comment by timbru31** [comment]
I've did some refactorings and now pass the result/success back to the JS promise.
Feel free to test the current master (`cordova plugin add https://github.com/timbru31/cordova-plugin-cordova-plugin-lottie-splashscreen.git`) and give me feedback. :)

I plan to release a new version soonish, too.

**Comment by timbru31** [comment]
This has been published as 0.8.0-beta.0
